docs(gen): tutorials requirements part 3 (#2933)
scaleway/docs-content: tutorials/k8s-velero-backup/index.mdx

    aws_secret_access_key=SCALEWAY_SECRET_KEY
    ```

> 3. Initialize Verlero. Make sure to replace `VELERO-BUCKET-NAME` with the name of the Object Storage bucket you want to use. Edit the backup location if your bucket is in a zone different from `fr-par`.

Reviewer: ```suggestion
3. Initialize Verlero. Make sure to replace `VELERO-BUCKET-NAME` with the name of the Object Storage bucket you want to use. Edit the backup location if your bucket is in a zone other than `fr-par`.
```
